### DISCUSSION TITLE
Cannot run podman remote build

### DISCUSSION THREAD

**Discussion by sirtow** [containers/podman#24058] [Q&A] [answered]
I have Ubuntu  VM with podman ,running on MacOS. The podman version is 4.3.1. 
I configured my MacOS podman to connect to that remote machine as user `dev`.
I'm trying to run `podman build` from my MacOS. it starts running but at some point it fails with :  

```
error running container: did not get container start message from parent: EOF
Error: building at STEP "RUN npm ci": netavark (exit code 1): setns: IO error: Operation not permitted (os error 1)
```
Running same build within the VM for same user `dev` fails with : 
```
rror: committing container for step {Env:[PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin NODE_VERSION=22.7.0 YARN_VERSION=1.22.22] Command:run Args:[npm ci] Flags:[] Attrs:map[] Message:RUN npm ci Original:RUN npm ci}: copying layers and metadata for container "2c50652f3f715041d8f3b0ea1728b5bccaa10ddd9c74747efaf186e91865d8f0": initializing source containers-storage:e21288c9a6cc-working-container: storing layer "36dceb5503e1c2a6d6203a0c57c83b3c9ff278155c832d91ac23a188afbe2a7f" to file: io: read/write on closed pipe
``` 

How do i fix it?
Thanks!

**Comment by Luap99** [comment, marked as answer]
Update your server to 5.X or downgrade you client to 4.3. In general it is not a good idea to mix the client/server versions, for the most part it works fine but sometimes there are bugs like that.

**Reply by sirtow** [reply]
Out of curiosity, where did you see my client version (it is indeed  5.0)?

**Reply by sirtow** [reply]
Im running Ubuntu 23.10:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 23.10
Release:	23.10
Codename:	mantic
``` 
Tried to update as per  https://podman.io/docs/installation#ubuntu, but it says its already on latest(`podman is already the newest version (4.3.1+ds1-8ubuntu1).`). 

I dont mind  to create a new VM  - Can you suggest Ubuntu version with latest 5.X  podman?
